shared/Cargo.toml

```diff
@@ -17,8 +17,8 @@ num-integer = "0.1"
 data-encoding = "2.3.2"
 data-encoding-macro = "0.1.12"
 lazy_static = "1.4.0"
-cid = { version = "0.8.5", default-features = false, features = ["serde-codec", "std"] }
-multihash = { version = "0.16.3", default-features = false, features = ["multihash-impl", "sha2", "sha3", "ripemd"] }
+cid = { version = "0.8.6", default-features = false, features = ["serde-codec", "std"] }
+multihash = { version = "0.16.2", default-features = false }
```

Comment:
Can you leave `features = ["multihash-impl", "blake2b"]` on multihash to avoid breaking `cargo test`? We have the dep in brave-core already.